Makefile

```
@@ -48,8 +44,13 @@ run: build ## Build and run binary
 fmt: ## gofmt and goimports all go files
 	find . -name '*.go' | while read -r file; do gofmt -w -s "$$file"; goimports -w "$$file"; done
 
-docker-build:  ## Build docker image
-	docker buildx build -o type=docker --network=host -t ${DOCKER_IMAGE_NAME} .
+docker-build:  ## Build docker image 
+	go generate ./...
+	docker buildx \
+	build -o type=docker \
+	--build-arg VERSION=${VERSION} \
+	--build-arg BUILD_TIME=${BUILD_TIME} \
+	--network=host -t ${DOCKER_IMAGE_NAME} .
```

Ident is a bit weird here IMO since flags are aligned with the command.

I'd suggest this, but do as you wish:
```suggestion
	docker buildx build
		-o type=docker \
		--build-arg VERSION=${VERSION} \
		--build-arg BUILD_TIME=${BUILD_TIME} \
		--network=host -t ${DOCKER_IMAGE_NAME} .
```